### PR TITLE
follow up upgrade to Qt 5.13+

### PR DESCRIPTION
**Pull request by 3nids**
this fixes value relation widget

also, going through JS with C++ variables is not supported anymore

for instance:
```
var modelIndex = myModel.index(row, col, ...)
var data = myModel.data(modelIndex, role)
```

is not working while

`var data = myModel.data(myModel.index(row, col, ...), role)
`
is working. But, Qt emits a warning:

Passing incompatible arguments to C++ functions from JavaScript is dangerous and deprecated.
This will throw a JavaScript TypeError in future releases of Qt!

therefore, it is advised to create helpers in the C++ classes directly:
`QVariant MyModel::dataFromRow(row, role)`

### CODE REVIEW

File: src/qml/RelationCombobox.qml [hunk 1]
```diff
@@ -32,7 +32,7 @@ Item {
     ComboBox {
       id: comboBox
 
-      property var currentValue: value
+      property var myCurrentValue: value
```
Comment (3nids) [inline src/qml/RelationCombobox.qml:35]: currentValue has been introduced as a readOnly property of ComboBox in Qt 5.14

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-921-39776c4412378068ed4f3c14f39fd1c473176092-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-921-39776c4412378068ed4f3c14f39fd1c473176092-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-921-39776c4412378068ed4f3c14f39fd1c473176092-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-921-39776c4412378068ed4f3c14f39fd1c473176092-x86_64.apk)

File: src/qml/FeatureForm.qml [hunk 2]
```diff
@@ -256,8 +256,8 @@ Page {
 
         Loader {
           id: attributeEditorLoader
-          asynchronous: true
-          visible: status == Loader.Ready
+          //asynchronous: true
```
Comment (signedav) [inline src/qml/FeatureForm.qml:259]: Remove or uncomment

File: src/qml/RelationCombobox.qml (hunk 1 above)
Comment (signedav) [reply src/qml/RelationCombobox.qml:35]: I don't like the "my" - maybe `comboBoxCurrentValue`

File: src/qml/FeatureForm.qml (hunk 2 above)
Comment (3nids) [reply src/qml/FeatureForm.qml:259]: nope, it's intentional

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-921-fab874eba1638f1de78dd21a552810b3f64ea5c7-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-921-fab874eba1638f1de78dd21a552810b3f64ea5c7-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-921-fab874eba1638f1de78dd21a552810b3f64ea5c7-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-921-fab874eba1638f1de78dd21a552810b3f64ea5c7-x86_64.apk)

**Pull request closed without merging**